src/orquestra/sdk/_base/_api/_wf_run.py

```diff
@@ -152,21 +151,9 @@ def start_from_ir(
             project_id: ID of the project for workflow - supported only on CE
 
         """
-        _config: RuntimeConfig
-        if isinstance(config, RuntimeConfig):
-            _config = config
-        elif isinstance(config, str):
-            _config = RuntimeConfig.load(config)
-        else:
-            raise TypeError(
-                f"'config' argument to `start_from_ir()` has unsupported "
-                f"type {type(config)}."
-            )
-        runtime: RuntimeInterface
-        if _config._runtime_name == "IN_PROCESS":
-            runtime = InProcessRuntime()
-        else:
-            runtime = _config._get_runtime()
+        _config = _resolve_config(config)
```

Comment:
**nitpick**: I know this was there before, but could you rename the `_resolve_config` to `resolve_config`? We shouldn't import private functions across modules.